services/brig/brig.cabal

```diff
@@ -434,6 +434,7 @@ executable brig-schema
       V57
       V58
       V59
+      V60
```

Comment:
akshay started the convention of adding descriptions to the migrations, a la `V60_Invitations_by_Email`.  I like that convention.

Reply:
:+1: